test/index.js

```diff
@@ -24,4 +24,19 @@ describe('PrismJS Configuration', () => {
             assert.equal(actual.trim(), expected.trim());
         });
     });
+
+    it(`should work with all languages`, () => {
+        const actual = transformSync('import Prism from "prismjs";', {
+            plugins: [
+                [plugin, { languages: 'all' }]
+            ],
+            babelrc: false
+        }).code;
+
+        // We expect an import for Prism core followed by 200-300 languages
+        if (!/^.+\n(?:\s*import "prismjs\/components\/prism-[\w-]+";){100,}\s*$/
+            .test(actual)) {
+            assert.fail();
+        }
```

Comment:
Instead of `if(! ... )` we can do `assert.ok(/^.+\n(?:\s*import "prismjs\/components\/prism-[\w-]+";){100,}\s*$/.test(actual)`

Reply:
Would it be possible to get all of the languages from `components.json` and assert we have the same number of lines (+ 1 for the core import)?

Reply:
This assumes that 1) `components.json` is somewhat stable (it's not) and 2) that languages only require other languages (I plan to move not-really-languages languages like Markup Templating into their own category eventually).

I mean, we could do it, but the test relies on unstable APIs.